File: C/2439.cpp
#include <stdio.h>
// Problem Number : 2439
// 첫째 줄에는 별 1개, 둘째 줄에는 별 2개, N번째 줄에는 별 N개를 찍는 프로그램. 
int main() {
    int n,i,j;
    scanf("%d", &n);
    for(i=1; i<=n; i++){
       for(j=n-i; j>0; j--)
           printf(" ");
    	for(j=0; j<i; j++)
             printf("*");
        printf("\n");
    } 
}

File: Java/_2439.java
package baekjoon;

import java.io.*;

/***
 Problem Name : 별 찍기 - 2
 Problem Number : 2349
 Problem Link : https://www.acmicpc.net/problem/2349
 ***/
public class _2439 {
    public static void main(String[] args) throws IOException {
        BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter out = new BufferedWriter(new OutputStreamWriter(System.out));

        int n = Integer.valueOf(in.readLine());

        for(int i=1; i<=n; i++) {
            for(int z=0; z<n-i; z++)
                out.write(" ");
            for(int j=n; j>n-i; j--)
                out.write("*");
            out.write("\n");
            out.flush();
        }

        out.close();
    }
}
